app/Filament/Resources/GameResource.php

```diff
@@ -542,6 +544,8 @@ public static function getPages(): array
             'create' => Pages\Create::route('/create'),
             'view' => Pages\Details::route('/{record}'),
             'edit' => Pages\Edit::route('/{record}/edit'),
+            'hubs' => Pages\Hubs::route('/{record}/hubs'),
+            'similar-games' => Pages\SimilarGames::route('/{record}/similar-games'),
```

Comment:
Out of curiosity: Why are these at the top navigation bar instead of in the lower navigation bar with achievements and such?

Related Hubs are in the lower navigation bar on individual hub pages.

Reply:
Only intention here is to give these a stable URL to link to. When they're relation managers on the bottom, they can be linked to via a query param, but I believe the link can change at any time if relation managers are added/removed to&from the resource.